documentation/advanced/tutorial-i18n-localization.asciidoc

```diff
@@ -170,10 +170,11 @@ public class MyLocale extends Div {
 }
 ----
 
-=== Supporting Right-to-Left mode in the application
+== Supporting Right-to-Left Mode
 
-Vaadin components have support for RTL languages, so to make your
-application handle this mode, a few changes are needed:
+Vaadin components have support for RTL languages. The components will work
+out-of-the-box on this mode, but to make your application handle this mode, a
```

Comment:
```suggestion
out-of-the-box in RTL mode, but to make your application switch to RTL, a
```

Reply:
I’d go with “make your application support both LTR and RTL modes”. Rolf’s suggestion sounds to me like it’s either or, you can’t support both with the same codebase.

Reply:
That's a valid point. It didn't sound like that to me, but I changed as suggested to make it more clear.